pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -524,6 +524,8 @@ def make_package(args):
 
     # Render out android manifest:
     manifest_path = "src/main/AndroidManifest.xml"
+    if args.extra_manifest_application_arguments:
+        args.extra_manifest_application_arguments = args.extra_manifest_application_arguments.strip("'")
```

Comment:
Looks like a duplicate from https://github.com/kivy/python-for-android/pull/2687, can we keep it separate from this PR?

Reply:
Yes, but I am a bit lost, @misl6 . Is the `develop` branch eventually get merged into `master` or not? How does that works?

Reply:
> Yes, but I am a bit lost, @misl6 . Is the `develop` branch eventually get merged into `master` or not? How does that works?

Uh, that may help: https://python-for-android.readthedocs.io/en/latest/contribute/#development-model

Reply:
Okay! Thanks a lot, @misl6 .
Then my PR makes no sense.
I already did some commits related to those arguments in the past to the `develop` branch, and was very surprised `service-class-name` and `activity-class-name` popped up again.
Personally, I am using a my own forks of `buildozer` and `p4a` for building my app.
Just wanted to feed back some fixes I found in the latest version.
Will close that PR and try to understand what is going on at the moment.

Reply:
Well `--network-security-config` and `--uses-cleartext-traffic` is something which is not present, so feel free to open another PR :)

Reply:
Yeah... those two arguments were in my first solution for that issue.

Then i just realized it makes no sense to create more and more different cmd line arguments. And decided to just create a new input file like that:
```
android:networkSecurityConfig="@xml/network_security_config"
android:usesCleartextTraffic="true"
android:requestLegacyExternalStorage="true"
```
and pass the path to that file to `buildozer`...

But it seems like it is better to go different way: pass the path to the file to `p4a` directly and there read the contents and inject it to the Android.xml template. There are some troubles with jinja2 templates and double quotes escaping...